src/web/views/FlowBuilder/sidePanel/properties/ActionSection.jsx

```diff
@@ -105,10 +105,10 @@ export default class ActionSection extends Component {
 
     return (
       <div>
-        <Panel collapsible defaultExpanded={true} header={header}>
+        <Panel collapsible="true" defaultExpanded={true} header={header}>
```

Comment:
should it be `{true}` instead?

Reply:
same below

Reply:
@emirotin , it it's `{true}`, I'm getting warning:

```
Warning: Received `true` for a non-boolean attribute `collapsible`.
If you want to write it to the DOM, pass a string instead: collapsible="true" or collapsible={value.toString()}.
```

Reply:
Oh, I thought it's the `Panel`'s prop, makes sense otherwise